server/package-lock.json

```diff
@@ -4,17 +4,6 @@
   "lockfileVersion": 1,
```

Comment:
I'm curious to know why the lockfile is not needed anymore 😃?

Reply:
What do you mean?

Reply:
Oh I thought it was removed but no it was just many `lines` removed not the whole file 😓 Sorry!